.github/workflows/pr.yml

```diff
@@ -28,10 +28,10 @@ jobs:
         name: "Checkout Che Dashboard source code"
         uses: actions/checkout@v2
       -
-        name: "Use Node 16"
+        name: "Use Node 14"
```

Comment:
Please revert this file

Reply:
We are still using  next:
```
      matrix:
        node-version: [14.x, 16.x]
```
(Node 14 and Node 16)

Ok. Let it be.

I changed that.